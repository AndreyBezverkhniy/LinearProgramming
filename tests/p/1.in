3
3
2 3 4
2 3 1 4
4 2 0 3
3 1 2 5

n # оригинальных переменных
m # уравнений
c1 .. cn
...
a{Bi}1 ... a{Bi}n <= b{Bi}
...

