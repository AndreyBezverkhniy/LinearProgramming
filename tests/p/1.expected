z = 4 + (0)*x0 + (0)*x1 + (3)*x2 + (-1)*x3 + (0)*x4 + (0)*x5
x0 = 2 + (0)*x0 + (-1.5)*x1 + (-0.5)*x2 + (-0.5)*x3 + (0)*x4 + (0)*x5
x4 = -5 + (0)*x0 + (4)*x1 + (2)*x2 + (2)*x3 + (0)*x4 + (0)*x5
x5 = -1 + (0)*x0 + (3.5)*x1 + (-0.5)*x2 + (1.5)*x3 + (0)*x4 + (0)*x5
